components/brave_wallet_ui/panel/container.tsx

```diff
@@ -79,37 +80,33 @@ import { TransactionStatus } from '../components/extension/post-confirmation'
-    isWalletLocked,
-    favoriteApps,
-    hasInitialized,
-    isWalletCreated,
-    networkList,
-    transactionSpotPrices,
-    activeOrigin,
-    defaultCurrencies,
-    transactions,
-    userVisibleTokensInfo,
-    defaultNetworks
-  } = useSelector(({ wallet }: { wallet: WalletState }) => wallet)
-  const {
-    connectToSiteOrigin,
-    panelTitle,
-    selectedPanel,
-    addChainRequest,
-    signMessageData,
-    switchChainRequest,
-    suggestedTokenRequest,
-    getEncryptionPublicKeyRequest,
-    decryptRequest,
-    connectingAccounts,
-    selectedTransaction,
-    hardwareWalletCode
-  } = useSelector(({ panel }: { panel: PanelState }) => panel)
+  const accounts = useUnsafeWalletSelector(WalletSelectors.accounts)
+  const activeOrigin = useUnsafeWalletSelector(WalletSelectors.activeOrigin)
+  const defaultCurrencies = useUnsafeWalletSelector(WalletSelectors.defaultCurrencies)
+  const defaultNetworks = useUnsafeWalletSelector(WalletSelectors.defaultNetworks)
+  const favoriteApps = useUnsafeWalletSelector(WalletSelectors.favoriteApps)
+  const hasInitialized = useSafeWalletSelector(WalletSelectors.hasInitialized)
+  const isWalletCreated = useSafeWalletSelector(WalletSelectors.isWalletCreated)
+  const isWalletLocked = useSafeWalletSelector(WalletSelectors.isWalletLocked)
+  const networkList = useUnsafeWalletSelector(WalletSelectors.networkList)
+  const selectedAccount = useUnsafeWalletSelector(WalletSelectors.selectedAccount)
+  const selectedNetwork = useUnsafeWalletSelector(WalletSelectors.selectedNetwork)
+  const selectedPendingTransaction = useUnsafeWalletSelector(WalletSelectors.selectedPendingTransaction)
+  const transactions = useUnsafeWalletSelector(WalletSelectors.transactions)
+  const transactionSpotPrices = useUnsafeWalletSelector(WalletSelectors.transactionSpotPrices)
+  const userVisibleTokensInfo = useUnsafeWalletSelector(WalletSelectors.userVisibleTokensInfo)
+  const connectToSiteOrigin = useUnsafePanelSelector(PanelSelectors.connectToSiteOrigin)
+  const panelTitle = useSafePanelSelector(PanelSelectors.panelTitle)
+  const selectedPanel = useSafePanelSelector(PanelSelectors.selectedPanel)
+  const addChainRequest = useUnsafePanelSelector(PanelSelectors.addChainRequest)
+  const signMessageData = useUnsafePanelSelector(PanelSelectors.signMessageData)
+  const switchChainRequest = useUnsafePanelSelector(PanelSelectors.switchChainRequest)
+  const suggestedTokenRequest = useUnsafePanelSelector(PanelSelectors.suggestedTokenRequest)
+  const getEncryptionPublicKeyRequest = useUnsafePanelSelector(PanelSelectors.getEncryptionPublicKeyRequest)
+  const decryptRequest = useUnsafePanelSelector(PanelSelectors.decryptRequest)
+  const connectingAccounts = useUnsafePanelSelector(PanelSelectors.connectingAccounts)
+  const selectedTransaction = useUnsafePanelSelector(PanelSelectors.selectedTransaction)
+  const hardwareWalletCode = useSafePanelSelector(PanelSelectors.hardwareWalletCode)
```

Comment:
Might be nice to have these grouped by `safe` and `unsafe` as well as `wallet` and `panel`! :)

Reply:
Updated. Grouped the selectors & added group comments